webservices/tasks/refresh.py

```diff
@@ -32,13 +32,31 @@ def refresh_and_rebuild():
     As with regular updates, email logs are also sent.
     """
     buffer = io.StringIO()
+    itemized_schedules = ['a', 'b']
+
     with mail.CaptureLogs(manage.logger, buffer):
         try:
-            manage.rebuild_aggregates()
-            manage.refresh_materialized()
+            projected_weekly_totals = manage.get_projected_weekly_itemized_totals(
+                itemized_schedules
+            )
+            manage.update_all()
```

Comment:
do we need to clear the old and new ques or is this taken care of in update_all()